content/references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/logrotate.md

```diff
@@ -1,7 +1,31 @@
 # logrotate
 [[Source]<br>](https://github.com/ripple/rippled/blob/743bd6c9175c472814448ea889413be79dfd1c07/src/ripple/rpc/handlers/LogRotate.cpp "Source")
 
-The `logrotate` command closes and reopens the log file. This is intended to help with log rotation on Linux file systems.
+The `rippled logrotate` command closes and reopens the log file. This is intended to help with log rotation on Linux file systems.
+
+Most linux systems come preinstalled with `logrotate`. Application specific log rotation scripts are placed in `/etc/logrotate.d`
+
+The following script is a sample that can be created as /etc/logrotate.d/rippled
+```
+/var/log/rippled/*.log {
+  daily
+  minsize 200M
+  rotate 7
+  nocreate
+  missingok
+  notifempty
+  compress
+  compresscmd /usr/bin/nice
+  compressoptions -n19 ionice -c3 gzip
+  compressext .gz
+  postrotate
+    /opt/ripple/bin/rippled --conf /opt/ripple/etc/rippled.cfg logrotate
+  endscript
+}
+```
+Parameters such as `minsize` and `rotate` can be changed depending on the `log_level` set in `rippled.cfg`. This sample script is based on standard `log_level` and will store approximately 2 weeks worth of logs in a compressed format.
```

Comment:
Suggest: "Assuming a default configuration that logs at the WARN level, this sample script would store approximately 2 weeks worth of logs, rolling every other day."

Reply:
Should we also note that a logrotate script will be installed by the ripple package for 1.3 and higher and that it can be modified (but does not need to be created)? Perhaps:  "For deb and rpm packages provided by ripple, this logrotate script will automatically be installed and can be modified as needed."

Reply:
I agree with these changes. Shall I make the modifications?